### PR TITLE
Convert UDP Server stuff to Kotlin

### PR DESCRIPTION
**Pull request by ImUrX**
Ignore the branch name >~>, should I remove any of the remaining deprecated objects?
Based on #647 
Requires SlimeVR/SolarXR-Protocol#109

### CODE REVIEW

**Comment by ButterscotchV** [comment]
This PR has a large number of formatting changes that will cause *many* conflicts with almost all of the open PRs based off of `tracker-rewrite`, please consider that before merging.

**Comment by ImUrX** [comment]
fixed :roll_eyes:

**Comment by ButterscotchV** [comment]
Noice, you can make format changes in a separate PR to make it easier to manage, thankyu <3

**Review by Eirenliel — CHANGES_REQUESTED** [review]

File: server/src/main/java/dev/slimevr/tracking/trackers/udp/SensorTap.kt
Comment (ImUrX) [reply server/src/main/java/dev/slimevr/tracking/trackers/udp/SensorTap.kt:11]: hm, you are right, i will fix that.

**Comment by 0forks** [comment]
Uri asked for a review, I can't add anything objective other than it works, I've tested with the actual trackers and owotrack.

Personally I find packet definitions in a single file instead of 20 easier to navigate, but the final word is on the maintainers anyway.

**Comment by ImUrX** [comment]
Ready for review & merge

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**
